### PR TITLE
style: change "starred" chats to "pinned"

### PR DESCRIPTION
**Pull request by suprko**
#### Relevant info and issue/PR links 
https://github.com/PeerioTechnologies/peerio-icebear/pull/117
https://app.clubhouse.io/peerio/story/1444/desktop-replace-star-with-pin 
https://app.clubhouse.io/peerio/story/2027/desktop-create-customicon-component-for-non-material-icons (note: `<CustomIcon>` component was created but has not been replaced in the rest of the app, just used in this particular story so far)

#### Testing instructions  
None needed IMO, this is just a restyling of previously-existing functionality.

----
### Repository owner

- [ ] Was tested and can be merged.
- [ ] PR name follows conventional changelog format.

[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)

### CODE REVIEW

File: src/ui-library/CustomIcon.jsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+class CustomIcon extends React.Component {
+    render() {
+        return (
+            <div className={css(
+                'p-custom-icon',
+                this.props.size,
+                this.props.className
+            )}>
+                <img src={`./static/custom-icons/${this.props.icon}.svg`} />
```
Comment (anri-asaturov) [inline src/ui-library/CustomIcon.jsx]: does it have to be an img inside div? can it be just one img?

**Comment by anri-asaturov** [comment]
Ready for testing

File: src/ui-library/CustomIcon.jsx (hunk 1 above)
Comment (suprko) [reply src/ui-library/CustomIcon.jsx]: I thought I had a reason for this but now I don't remember. Testing just  now, I think it'll be the same

**Comment by seavan** [comment]
@suprko is this tested?

**Comment by suprko** [comment]
@seavan I tested it, if that's what you mean! I don't think it needs to go to QA. Just a 1-button function that already existed, only styling is different.

**Comment by anri-asaturov** [comment]
Well then give me a signal to merge ;)

**Pull request merged**